packages/app-bridge/src/react/useGuidelineActions.ts

```diff
@@ -286,35 +290,37 @@ export const useGuidelineActions = (appBridge: AppBridgeTheme) => {
         [appBridge],
     );
 
-    const deletePage = useCallback(
-        async ({ id, documentId }: { id: number; documentId: number }) => {
+    const deleteDocumentPage = useCallback(
+        async ({ id, documentId, categoryId }: { id: number; documentId: number; categoryId: Nullable<number> }) => {
```

Comment:
documentId, and categoryId should be optional? Or categoryId at least. Depending on what BE requires.

Reply:
We need to type guard it as well